src/Native/System.Net.Security.Native/pal_gssapi.cpp

```diff
@@ -227,12 +227,12 @@ extern "C" uint32_t NetSecurityNative_ReleaseCred(uint32_t* minorStatus, GssCred
     return gss_release_cred(minorStatus, credHandle);
 }
 
-extern "C" void NetSecurityNative_ReleaseGssBuffer(void* buffer, uint64_t length)
+extern "C" void NetSecurityNative_ReleaseGssBuffer(void* buffer, size_t length)
```

Comment:
This should remain uint64_t and then be static_cast to size_t (an assert could also be added to verify that it won't overflow).  Making this size_t will break the interop between the managed and native code, as size_t can change size based on platform.  The closest thing available in managed code would be UIntPtr, but even that's not technically correct, as there's no guarantee in the spec that size_t will be 32-bit on all 32-bit platforms... it needs only be large enough to hold 32-bit, but could theoretically be larger.  That's partially why all of our signatures that work with a size_t or ssize_t take a 64-bit value, instead; the other reason is that it's much easier to work with on the managed side.